package.json

```diff
@@ -62,14 +62,14 @@
 		"jest-image-snapshot": "3.0.1",
 		"matchdep": "~2.0.0",
 		"prettier": "npm:wp-prettier@2.0.5",
-		"qunit": "~2.19.1",
+		"qunit": "~2.19.4",
 		"react-refresh": "0.10.0",
-		"sass": "^1.55.0",
-		"sinon": "~14.0.1",
-		"sinon-test": "~3.1.4",
+		"sass": "^1.57.1",
+		"sinon": "~14.0.2",
```

Comment:
It looks like `15.0.1` is the latest version. Can we update to that version here? It looks like only minor changes that should not cause problems.

Reply:
Sure, It can update to that version here.